### ISSUE TITLE
Import  ImportThisFileIntoYourWirtBotInterface.json does not do anything 

### ISSUE THREAD

**Issue by agalewsky-pnn** [b-m-f/WirtBot#108]
I have successfully install wirtbot and have connected to the dashboard. However, when I try to import  ImportThisFileIntoYourWirtBotInterface.json as a backup and click the import button - nothing happens.  I can't find any errors thrown.  I have tried Chrome, Firefox and Safari and they all yield the same results.

Any thoughts?

-Andy

**Comment by b-m-f** [comment]
Hi @agalewsky-pnn / Andy,

Nothing shows up after a refresh either?

Could you paste the output of `ImportThisFileIntoYourWirtBotInterface.json` ?

Just make sure to remove the Keys from the output :)

**Comment by b-m-f** [comment]
So, I just set up a new WirtBot again and could not reproduce this. 

Importing worked without any issues. Are you trying to import on `wirtbot.wirt.internal`?

**Comment by agalewsky-pnn** [comment]
Let me try again.  Is it important to do it from that address?

**Comment by b-m-f** [comment]
Hm, the address should not matter too much.
I am just curious how else you would reach the DashBoard?

**Comment by agalewsky-pnn** [comment]
I used the IP address, but I think I also tried the DNS name as well.   One
thing I am going to check if I have any popup blockers, etc. engaged on my
browser(s)

**Comment by b-m-f** [comment]
@agalewsky-pnn Make sure to access via the internal IP or the Hostname.

There was an uncommented line in the installer. This is fixed now.

Try `npm install -g @wirtbot/installer@1.0.8`. And use the update functionality to get the latest version

**Comment by agalewsky-pnn** [comment]
I am getting this error in the browser console:
Uncaught SyntaxError: Unexpected token
 in JSON at position 175
    at JSON.parse (<anonymous>)
    at FileReader.t.onload (Settings.vue:93)

I am using the file generated by the application

-Andy

**Comment by b-m-f** [comment]
Interesting. 

Could you post your JSON as well please?

Just remove the keys beforehand

**Comment by b-m-f** [comment]
2 more things:

- Have you updated the latest `@wirtbot/installer`?
- Did you already use the `update` functionality of the installer?

**Comment by b-m-f** [comment]
No more info given -> closed
